composer.json

```diff
@@ -21,13 +21,17 @@
         "symfony/phpunit-bridge": "^2.7 || ^3.0",
         "propel/propel1": "^1.6",
         "fabpot/php-cs-fixer": "~0.1 || ^1.0",
-        "sllh/php-cs-fixer-styleci-bridge": "^2.0"
+        "sllh/php-cs-fixer-styleci-bridge": "^2.0",
+        "knplabs/knp-snappy": "^0.4",
+        "h4cc/wkhtmltopdf-amd64": "^0.12"
     },
     "suggest": {
         "ext-curl": "*",
         "symfony/routing": "^2.3 || ^3.0",
         "symfony/property-access": "^2.3 || ^3.0",
-        "propel/propel1": "^1.6"
+        "propel/propel1": "^1.6",
+        "knplabs/knp-snappy": "^0.4",
```

Comment:
Please add a text information, not a version here.

See https://getcomposer.org/doc/04-schema.md#suggest